src/cache.rs

```diff
@@ -1,10 +1,17 @@
+// cacache uses asyncd-std which has no wasm support, so currently no caching support on wasm
```

Comment:
Could we instead (or in addition) avoid calling into `cache` at all when `target_family = "wasm"`?

Reply:
done; i'd keep compiling cache.rs on wasm though because caching there should be possible (either in cwd like pytest or to some extent in local storage), it's just cacache that is inactive and hasn't completed tokio support (https://github.com/zkat/cacache-rs/pull/29)